man/adoc/bpftrace.adoc

```diff
@@ -188,6 +188,17 @@ Print bpftrace version information.
 Enable verbose messages.
 For more details see the <<Verbose Output>> section.
 
+=== Program Options
+
+You can also pass custom options to a bpftrace program/script itself via positional or named parameters.
+All positional and named parameters should come after a double dash e.g.
```

Comment:
... should come after the script filename or a double dash if running an inline script with `-e`